### DISCUSSION TITLE
Should this Package support historic data?

### DISCUSSION THREAD

**Discussion by Martin-Welte** [spatie/holidays#46] [General]
## Why?

Most countries changed their holidays since the 1970. 
Should this Package support changes in Holidays like the change in the Netherlands from Queensday to Kingsday which switched the date based on the current monarchs Birthday? 

## Issue: Non Existent Countries

Another issue would be non-existent countries in a specific year. 
South Sudan comes to mind. (independence in 2011)

**Comment by Nielsvanpach** [comment]
It would be a nice contribution to make sure historical data is correct! Feel free to PR this for NL.

**Comment by inDeev** [comment]
Should? No, it must. This package is made to support dates since 1970, so each year for each country must return valid data for that year. It's quite easy to implement like I did it in [PR:159](https://github.com/spatie/holidays/pull/159), only research takes some time.

**Comment by Martin-Welte** [comment]
@inDeev Thanks for the link to your PR. I was aware of the existing tests, but since historic data was not implemented for the initially commited countries I thought it might be out of scope for this package.
